### PR TITLE
Added finnish dvorak layout

### PR DESCRIPTION
**Pull request by keisari-kumkvatti**

### CODE REVIEW

**Comment by colmbuckley** [comment]
Please rename `LISENCE.TXT` to `LICENSE` and `README.TXT` to `README` for consistency with other layouts.

**Comment by keisari-kumkvatti** [comment]
> Please rename `LISENCE.TXT` to `LICENSE` and `README.TXT` to `README` for consistency with other layouts.

Renamed.

**Comment by colmbuckley** [comment]
Sorry, still not quite correct - should be `LICENSE` rather than `LISENCE`. English is a dumb language, I know…

**Comment by keisari-kumkvatti** [comment]
> Sorry, still not quite correct - should be `LICENSE` rather than `LISENCE`. English is a dumb language, I know…

My bad, it should be corrected now.

File: finnish-dvorak/manifest.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    {
+      "name": "Finnish dvorak",
+      "type": "ime",
+      "id": "dvorak_fi",
+      "description": "This enables the Finnish Dvorak layout in Chrome OS",
+      "language": ["fi-FI"],
+      "layouts": ["fi(fidvorak)"]
+    }
+  ]
+}
```
Comment (colmbuckley) [inline finnish-dvorak/manifest.json:19]: Please add trailing newline.

Comment (keisari-kumkvatti) [reply finnish-dvorak/manifest.json]: Did it.

**Pull request merged**